components/cam/cime_config/testdefs/testmods_dirs/cam/thetahy_sl_pg2/user_nl_cam

```diff
@@ -0,0 +1,17 @@
+theta_hydrostatic_mode = .true.
+se_fv_phys_remap_alg = 1
+tstep_type = 5
+theta_advect_form = 1
+hypervis_subcycle_tom = 1
+hypervis_subcycle = 1
+transport_alg = 12
+semi_lagrange_cdr_alg = 20
```

Comment:
why not 21? is 21 the most advanced now?

Reply:
Good question. 21 is a bit of a pet project for me. I want it available for future work. But for now, I want to stick with the tried-and-true 20.